debian/README

```diff
@@ -0,0 +1,84 @@
+# llmnrd - Link-Local Multicast Resolution Daemon
```

Comment:
Is it really necessary to copy the README to debian/README in order to install it? I'd much prefer to avoid duplicating it.